fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerSimpleTest.java

```diff
@@ -81,6 +85,25 @@
  * build full indexes. ({@link #testConfigLoader()} does use a such API but it is not necessary.)
  */
 public class OnlineIndexerSimpleTest extends OnlineIndexerTest {
+    private static final Pattern BATCH_GRV_PATTERN = eventCountPattern(FDBStoreTimer.Events.BATCH_GET_READ_VERSION);
+    private static final Pattern SCAN_RECORDS_PATTERN = eventCountPattern(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RECORDS_SCANNED);
+    private static final Pattern BUILD_RANGES_PATTERN = eventCountPattern(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RANGES_BY_COUNT);
+
+    private static Pattern eventCountPattern(StoreTimer.Event event) {
+        return Pattern.compile(".*" + event.logKeyWithSuffix("_count") + "=\"(\\d+)\".*");
+    }
+
+    private static int extractCount(Pattern pattern, String logEvent) {
+        Matcher matcher = pattern.matcher(logEvent);
+        assertTrue(matcher.matches(), () -> String.format("expected \"%s\" to have pattern \"%s\"", logEvent, pattern));
+        return Integer.parseInt(matcher.group(1));
+    }
+
+    private static void assertAbsent(Pattern pattern, String logEvent) {
+        Matcher matcher = pattern.matcher(logEvent);
+        assertFalse(matcher.matches(), () -> String.format("did not expect \"%s\" to have pattern \"%s\"", logEvent, pattern));
+    }
+
```

Comment:
I'd put them there initially because these are the only online indexing tests that look at log events. But I could see these being more useful for other tests that want to look at metrics in log events, so I've moved them to `TestHelpers`